File: core.shared/Net/DTO/V1/Remboursement/RemboursementPrevoyance.cs
using Core.Shared.Net.DTO.V1.Beneficiaire;

namespace Core.Shared.Net.DTO.V1.Remboursement
{
    public class RemboursementPrevoyance
    {
        public DateTime DateDeRemboursementComplet { get; set; }
        public string NumeroParticipant { get; set; }
        public string Identifiant_Decompte { get; set; }
        public string TruncatDateDeRemboursement { get; set; }
        public string DateReglement { get; set; }
        public string DateDeRemboursement { get; set; }
        public string Libelle { get; set; }
        public string Montant { get; set; }
        public DateTime DateIndeminisationMin { get; set; }
        public DateTime DateIndeminisationMax { get; set; }
        public IList<LeBeneficiaire> Beneficiaires { get; set; }
        public IList<CodesActes> Codactes { get; set; }
        public string TypeRemboursement { get; set; }
        public string LibelleTypeRemboursement { get; set; }
        public string PeriodeDateRemboursement { get; set; }
        public string NumeroDeDossier { get; set; }
        public string TauxActivite { get; set; }
        public string MontantBrut { get; set; }
        public string CsgNonImposable { get; set; }
        public string CsgImposable { get; set; }
        public string RdsImposable { get; set; }
        public string CasaImposable { get; set; }
        public string MontantNetFiscal { get; set; }
        public string PasTypeTaux { get; set; }
        public string PasTaux { get; set; }
        public string PasMontant { get; set; }
        public string Civilite { get; set; }
        public bool Pdf { get; set; }
        public IList<RemboursementPrevoyanceDetails> RemboursementsPrevoyanceDetail { get; set; }
    }
}

File: core.shared/Typescript/DTO/V1/Remboursement/remboursementPrevoyance.ts
/* Auto Generated */

import { LeBeneficiaire } from "./../Beneficiaire/leBeneficiaire";
import { CodesActes } from "./codesActes";
import { RemboursementPrevoyanceDetails } from "./remboursementPrevoyanceDetails";

export interface RemboursementPrevoyance {
    dateDeRemboursementComplet: Date;
    numeroParticipant: string;
    identifiant_Decompte: string;
    truncatDateDeRemboursement: string;
    dateReglement: string;
    dateDeRemboursement: string;
    libelle: string;
    montant: string;
    dateIndeminisationMin: Date;
    dateIndeminisationMax: Date;
    beneficiaires: LeBeneficiaire[];
    codactes: CodesActes[];
    typeRemboursement: string;
    libelleTypeRemboursement: string;
    periodeDateRemboursement: string;
    numeroDeDossier: string;
    tauxActivite: string;
    montantBrut: string;
    csgNonImposable: string;
    csgImposable: string;
    rdsImposable: string;
    casaImposable: string;
    montantNetFiscal: string;
    pasTypeTaux: string;
    pasTaux: string;
    pasMontant: string;
    civilite: string;
    pdf: boolean;
    remboursementsPrevoyanceDetail: RemboursementPrevoyanceDetails[];
}
